capy/src/main/java/com/jocmp/capy/accounts/reader/ReaderCredentials.kt

```diff
@@ -1,24 +1,30 @@
 package com.jocmp.capy.accounts.reader
 
+import com.jocmp.capy.ClientCertManager
 import com.jocmp.capy.accounts.Credentials
 import com.jocmp.capy.accounts.Source
 import com.jocmp.capy.accounts.baseHttpClient
+import com.jocmp.capy.accounts.reader.ReaderOkHttpClient.clientCertAlias
 import com.jocmp.readerclient.GoogleReader
 import com.jocmp.readerclient.GoogleReader.Companion.UNAUTHORIZED_MESSAGE
 
 data class ReaderCredentials(
     override val username: String,
     override val secret: String,
     override val url: String,
+    override val clientCertAlias: String,
```

Comment:
per my [other comment](https://github.com/jocmp/capyreader/pull/688/files#r2314600853), the manager can slot in here:

```suggestion
    override val clientCertAlias: String,
    private val clientCertManager: ClientCertManager
```